src/cards_window.cr

```diff
@@ -43,4 +44,29 @@ class CardsWindow < OptionSelectWindow
     @win.erase
     @win.refresh
   end
+
+  def add_helps(win)
+    win.add_help(key: "c", description: "Create a new card on this list")
+    super
+  end
+
+  def handle_key(key)
+    case key
+    when 'c'
+      contents = JSON.build do |json|
+        json.object do
+          json.field "name", ""
+          json.field "description", ""
+        end
+      end
+
+      Editor.run(contents: contents) do |card_json|
+        card = JSON.parse(card_json)
+        API.post("/cards", params: "idList=#{@list_id}", form: "name=#{card["name"]}&desc=#{card["description"]}")
```

Comment:
This works for now, but we'll need a better way of building this URL so that certain characters are escaped.